valmlord/gulpify
# Feature/fonts tasks

## Description
This pull request introduces new tasks to handle font conversion and specifically adds support for the WOFF2 font format. These changes enhance the project's font management capabilities.

## Changes
### Commit 1: Added a task for converting fonts
- Created a new Gulp task to handle font conversion.
- Configured the task to process font files (e.g., TTF, OTF) and generate additional formats (e.g., WOFF).

### Commit 2: Add WOFF2 conversion
- Added support for the WOFF2 font format by integrating the necessary tools  `gulp-ttf2woff2`.
- Updated the font conversion task to include WOFF2 output.

## How to Test
2. Place font files TTF, OTF in the appropriate directory `app/assets/fonts`.
3. Execute the relevant Gulp task `gulp buildFonts` to convert the fonts.
4. Verify that the converted font files (including WOFF2) are correctly generated in the `build/assets/fonts` directory.

## Additional Notes
- This update aims to improve font compatibility and performance by providing WOFF2 support.
